src/Objects/EventSubSignature.php

```diff
@@ -54,15 +49,11 @@ private static function getTimestamp(?string $rawTimestamp): ?int
             return null;
         }
 
-        if ( ! preg_match(self::TIMESTAMP_PATTERN, $rawTimestamp, $matches)) {
-            return null;
+        if (preg_match('/\.[0-9]{9}Z$/', $rawTimestamp)) {
+            $rawTimestamp = substr_replace($rawTimestamp, '', -2, 1);
         }
```

Comment:
```suggestion
        if (preg_match('/\.([\d]{9,})Z$/', $rawTimestamp, $match)) {
            $length = strlen($match[1]) - 8;
            $rawTimestamp = substr_replace($rawTimestamp, '', ($length * -1) - 1, $length);
        }
```

This will also match timestamps with fraction lengths > 9. Apparently Twitch rolls the dice on how much fraction characters will be sent